arrays.dd

```diff
@@ -346,16 +346,17 @@ a[] = b[] + 4;
 ---
 
         $(P A vector operation is indicated by the slice operator appearing
-        as the lvalue of an =, +=, -=, *=, /=, %=, ^=, $(AMP)= or |= operator.
-        The rvalue can be an expression consisting either of an array
-        slice of the same length and type as the lvalue or an expression
-        of the element type of the lvalue, in any combination.
+        as the left-hand side of an =, +=, -=, *=, /=, %=, ^=, $(AMP)= or |=
+        operator.
+        The right-hand side can be an expression consisting either of an array
+        slice of the same length and type as the left-hand side or an expression
+        of the element type of the left-hand side, in any combination.
         The operators supported for vector operations are the binary
         operators +, -, *, /, %, ^, $(AMP) and |, and the unary operators
         - and ~.
         )
 
-        $(P The lvalue slice and any rvalue slices must not overlap.
+        $(P The slice on the left and any slices on the right must not overlap.
```

Comment:
Would you be ok with the first one, too? I think it's worth acknowledging that there may not be any slices on the right.